src/cairoUtilFuncGen/memory/memoryStruct.ts

```diff
@@ -84,12 +84,12 @@ export class MemoryStructGen extends StringIndexedFuncGen {
       code: [
         `func ${funcName}{range_check_ptr, warp_memory: DictAccess*}(${argString}) -> (res:felt){`,
         `    alloc_locals;`,
-        `    let (start) = wm_alloc(${uint256(structType.width)});`,
+        `    let (__warp_start) = wm_alloc(${uint256(structType.width)});`,
```

Comment:
I think a better solution here wouldn't be mangling our vars in generated functions but mangling input parameters. In this case like instead of just using the name of the struct, have like member_structname or smth like that